src/components/main/Team.tsx

```diff
@@ -0,0 +1,25 @@
+import Card from "../UI/Card"
+
+const Team = () => {
+    const teamMembers = [
```

Comment:
we need to work on code, and make it cleaner,
the array can be defined outside the function/component, maybe even in an util file, import it from there and just loop to render

Reply:
Also Team, should render a TeamMember component that takes as props the (url, name, title, bio)

Reply:
Ok. It's done.